shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -63,11 +63,76 @@ mod keyspace {
         );
     }
 
-    pub fn test(session: Session) {
-        test_create_keyspace(&session);
-        test_use_keyspace(&session);
-        test_drop_keyspace(&session);
-        test_alter_keyspace(&session);
+    pub fn test(session: &Session) {
+        test_create_keyspace(session);
+        test_use_keyspace(session);
+        test_drop_keyspace(session);
+        test_alter_keyspace(session);
+    }
+}
+
+mod table {
+    use cassandra_cpp::Session;
+
+    use crate::cassandra_int_tests::{
+        assert_query_result, assert_query_result_contains_row,
+        assert_query_result_not_contains_row, run_query, ResultValue,
+    };
+
+    fn test_create_table(session: &Session) {
+        run_query(
+            session,
+            "CREATE TABLE test_table_keyspace.my_table (id UUID PRIMARY KEY, name text, age int);",
+        );
+        assert_query_result_contains_row(
+            session,
+            "SELECT table_name FROM system_schema.tables;",
+            &[ResultValue::Varchar("my_table".into())],
+        );
+    }
+
+    fn test_drop_table(session: &Session) {
+        run_query(
+            session,
+            "CREATE TABLE test_table_keyspace.delete_me (id UUID PRIMARY KEY, name text, age int);",
+        );
+
+        assert_query_result_contains_row(
+            session,
+            "SELECT table_name FROM system_schema.tables;",
+            &[ResultValue::Varchar("delete_me".into())],
+        );
+        assert_query_result(
+            session,
+            "DROP TABLE IF EXISTS test_table_keyspace.delete_me;",
```

Comment:
should we use IF EXISTS here?
It would indicate something went very wrong if the table didnt exist.